### ISSUE TITLE
perform_extraction: data_smooth is unused

### ISSUE THREAD

**Issue by schoffelen** [agramfort/eeglab-plugin-ieee-tbme-2010#1]
Hi Alex,
Was it your intention to build in an option for smoothing, without the smoothing to have an effect?
Probably the input into gc_lags should be data_smooth, rather than points_ordered.
Also, if movav is to be taken seriously, the input (and output) should probably not be transposed.

**Comment by agramfort** [comment]
@schoffelen I have not looked at this code in ages... does it run for you? do you have empirical results that look suspicious?

**Comment by schoffelen** [comment]
No, it seems to work just fine, provided the single trial data is of good quality, so MEG virtual channel works, normal sensor: no.

I have written a fieldtrip function that taps into the plugin, and it gives reasonable results (e.g. giving latencies in motor-related virtual channels that highly correlates with reaction times.

Well done!

**Comment by schoffelen** [comment]
The only thing is that the smoothing is not applied.

**Comment by agramfort** [comment]
really glad to hear this !

don't hesitate to send a pull request if you think something should be
fixed.
You probably know this code better than me by now :)

>
